go.mod

```diff
@@ -3,12 +3,11 @@ module github.com/vmware/terraform-provider-vcf
 go 1.19
 
 require (
-	github.com/davecgh/go-spew v1.1.1
 	github.com/go-openapi/runtime v0.26.0
 	github.com/go-openapi/strfmt v0.21.7
 	github.com/hashicorp/terraform-plugin-docs v0.14.1
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.26.1
-	github.com/vmware/vcf-sdk-go v0.1.0
+	github.com/vmware/vcf-sdk-go v0.1.1
```

Comment:
There is an open PR for this. See #17

Reply:
A manual intervention is required since the method names, provided by the SDK have changed "GETHTTP... -> GetHTTP...".